### ISSUE TITLE
npm verison can't find type declarations for bitcore-lib

### ISSUE THREAD

**Issue by aechan** [poetapp/poet-js#8]
The version pulled from npm can't find the type definition for the `bitcore-lib` module. This makes it so a TypeScript project using poet-js won't build because the compiler throws the error:
```
dist/babel/src/Bitcoin.d.ts(2,26): error TS7016: Could not find a declaration file for module 'bitcore-lib'.
```
But when I pull this repo from git and build it with `npm run build` everything works and compiles properly.

I'm not sure if this is an error on my part or if the actual type declaration file for bitcore-lib isn't being included in the npm published version.

**Comment by aechan** [comment]
Creating the directory `node_modules/@types/bitcore-lib` and copying `bitcore-lib.d.ts` from this repo's `typings` directory and renaming it `index.d.ts` then placing it in `node_modules/@types/bitcore-lib/index.d.ts` and creating a `package.json` with `name: @types/bitcore-lib` allows a TypeScript project using poet-js to compile and run properly. 

I tried placing `bitcore-lib.d.ts` in a few other places, but the only place the TypeScript compiler could seem to find it was `@types/bitcore-lib`

**Comment by lautarodragan** [comment]
Hey @aechan, thanks for reporting this and going through all that work.

Indeed this looks like a bug. Looking at [unpkg/poet-js](https://unpkg.com/poet-js@1.0.1/dist/babel/src/) I can see that the typings are missing. They are actually being [ignored in .npmignore](https://github.com/poetapp/poet-js/blob/master/.npmignore#L3). I'm not sure how this came up like this. I'm thinking it's working for the Node "out of luck", because the Node has this same file in it.

I think the best approach would be to publish these typings as a separate package and make them a dependency on both projects. I'll try to sort that out quickly.

**Comment by lautarodragan** [comment]
Now waiting for https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23547...

**Comment by lautarodragan** [comment]
@aechan, just published poet-js@1.0.3 which should fix this issue. 

Please let me know if it works for you now.

**Comment by lautarodragan** [comment]
Closing this for now. Let me know if you have any other issues @aechan
